### DISCUSSION TITLE
ExifTag name for ISO value

### DISCUSSION THREAD

**Discussion by SLS8** [dlemstra/Magick.NET#822] [General]
Hi
I was trying to read a few standard parameters from jpeg files (ISO, Exposure etc) using GetValue() in Magick.NET 7.22.2.2 but I had problems with the ISO value. First I tried "ExifTag.ISOSpeed" but it returns null.
Using "identify" in command line I see that the tag containing ISO value is named "PhotographicSensitivity" but that tag doesn't appear in ExifTag class.
After some searching in the ExifTool program help page, I find that ISO was also stored in an array "ISOSpeedRatings" for EXIF 2.2 (and I could read it from there in magick.net) and under "PhotographicSensitivity" for EXIF 2.3 

So in case someone else has problems finding how to read the ISO value maybe the tag "PhotographicSensitivity" should be added to ExifTag class to match the imagemagick identify naming.

**Comment by dlemstra** [comment]
Can you share the input image so I can test this locally?

**Comment by SLS8** [comment]
Sure, you can use the dunes sample image from here photographylife.com/what-is-exif-data

_./magick.exe identify -format "%[EXIF:*]"_ shows the ISO value as _exif:PhotographicSensitivity=100_

In magick.net i find an "100" value using _.GetValue(ExifTag.ISOSpeedRatings).Value[0]_ 
But _ISOSpeed_ tag is null and there's no _PhotographicSensitivity_ tag

**Comment by dlemstra** [comment]
You can use `ExifTag.ISOSpeedRatings` instead (https://www.awaresystems.be/imaging/tiff/tifftags/privateifd/exif/isospeedratings.html).
